test/Libraries/Microsoft.Extensions.Http.Diagnostics.Tests/Microsoft.Extensions.Http.Diagnostics.Tests.csproj

```diff
@@ -26,6 +26,7 @@
     <PackageReference Include="Microsoft.Extensions.Http" />
     <PackageReference Include="Newtonsoft.Json" />
     <PackageReference Include="System.Net.Http" Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net6.0'))" />
+    <PackageReference Include="System.Security.Cryptography.Xml" Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net6.0'))" />
```

Comment:
Is this intended? Was it in order to address an issue flagged by SDL? If so, should this be in the rest of the branches too?

Reply:
Yes, it is intended, and it was flagged by the compliance tools on the release branches.